packages/lambdas/package-lock.json

```diff
@@ -2773,21 +2773,21 @@
       }
     },
     "node_modules/@metriport/api-sdk": {
-      "version": "7.1.4",
-      "resolved": "https://registry.npmjs.org/@metriport/api-sdk/-/api-sdk-7.1.4.tgz",
-      "integrity": "sha512-5kg91UV3OyJ+EX3q8HWABjdc1obHzR80KJsk1CkxOAFIJAhl0fZgH79rmCMe9J8yHaLY+iJy/U1kNF5jKQt2Kg==",
+      "version": "7.1.3",
```

Comment:
Why are we downgrading this and the other libs on this file?

AFAIK we shouldn't be updating the lambdas' lock if no change had been done on it and/or its dependencies, right?